accounts/templates/accounts/participant_detail.html

```diff
@@ -82,15 +83,15 @@
                         {% for birthday in user.latest_demographics.child_birthdays %}
                             {{ birthday | timesince|default:"N/A" }}
                             {% if not forloop.last %};{% endif %}
-                        {% empty %}{% trans "No Response" %}
+                        {% empty %}
+                            {% trans "No Response" as trans_no_response %}
+                        {% empty_text trans_no_response %}
```

Comment:
I think we should go back to the non-template-tag version in this case, i.e. `<td>No Response</td>` to match the other NA values in the rest of the table.